tests/integration_tests/functional/test_metrics.py

```diff
@@ -370,3 +373,36 @@ def test_net_metrics(test_microvm_with_api):
         # test that we get metrics while interacting with different interfaces
         net_metrics.validate(test_microvm)
         assert exit_code == 0
+
+
+def test_block_metrics(test_microvm_with_api):
+    """
+    Validate that NetDeviceMetrics doesn't have a breaking change
+    and "net" is aggregate of all "net_*" in the json object.
+    """
+    test_microvm = test_microvm_with_api
+    test_microvm.spawn()
+
+    # Add first scratch block device.
+    fs1 = drive_tools.FilesystemFile(
+        os.path.join(test_microvm.fsfiles, "scratch1"), size=128
+    )
+    test_microvm.add_drive("scratch1", fs1.path)
+
+    # Set up a basic microVM.
+    # (this is the second block device added).
+    test_microvm.basic_config()
```

Comment:
any reason for `add_drive` -> `basic_config` -> `add_drive` sequence vs `basic_config` -> 2 x `add_drive`, for instance?

Reply:
This sequence was picked form this sequence was used in test_drives.py where it meant testing the rootfs comes first however, in this metrics context there is no specific reason. Just a copy paste of steps.